doc/nrf/known_issues.rst

```diff
@@ -1235,6 +1242,13 @@ NCSDK-15707: Visual glitches when updating an RGB LED's color in :ref:`caf_leds`
   **Workaround:** Make sure your application includes the devicetree overlay file in which PWM period is decreased.
   For example, include the following commit to solve the issue for the :ref:`nrf_machine_learning_app` application for Nordic Thingy:53: ``fa2b57cddbaacf393c77def5d0302e1a45138d21``.
 
+.. rst-class:: v2-1-0
+
+NCSDK-16644: Sensor manager macro incorrectly converts float to sensor value
+  Sensor manager macro ``FLOAT_TO_SENSOR_VALUE`` converts float to sensor value incorrectly.
```

Comment:
:c:macro:`FLOAT_TO_SENSOR_VALUE`

Reply:
`Sensor manager macro :c:macro:`FLOAT_TO_SENSOR_VALUE` converts float to sensor value incorrectly, because of missing brackets around macro argument.`

Reply:
I would keep this as backticks as the macro don't have an entry in the API documentation and will not link.